packages/components/src/components/Divider/Divider.tsx

```diff
@@ -17,7 +17,7 @@ export type DividerProps = {
      * The style of the divider
      * @default "solid"
      */
-    style?: DividerStyle;
+    variant?: DividerStyle;
```

Comment:
Sorry I just saw it now with CTRL F 😬 
Could you update the type name to `DividerVariant` (same in other files) and also the JSDoc (same in other files)